brayns/common/material/Material.h

```diff
@@ -66,7 +66,9 @@ class Material
     BRAYNS_API TextureTypes& getTextures() { return _textures; }
     BRAYNS_API void setTexture(const TextureType& type,
                                const std::string& filename);
-
+    BRAYNS_API void lock() { _locked = true; }
+    BRAYNS_API void unlock() { _locked = false; }
+    BRAYNS_API bool locked() const { return _locked; }
```

Comment:
please put a doxygen here 'cause I already forgot what this was for again

Reply:
Doc added